### ISSUE TITLE
Marker Labeling Mode

### ISSUE THREAD

**Issue by dantist90** [jeromeetienne/AR.js#599]
Hello. How to change marker border color? From black to white?

Ar.js file contain function "setLabelingMode" and comment:
"ARToolKit's labelling algorithm can work with both black-bordered markers on a white background (AR_LABELING_BLACK_REGION) or white-bordered markers on a black background (AR_LABELING_WHITE_REGION)."

I use "a-frame" for ar.js
Maybe you have example with a white marker? Or documentation? Thanks!

**Comment by nicolocarpignoli** [comment]
You cannot use a white color as border. It has to be a dark one, and with high contrast with the background of the marker.

To change that, you could create your own pattern with this tool (on the left, you can change border color from black to another darker one)

https://jeromeetienne.github.io/AR.js/three.js/examples/marker-training/examples/generator.html

**Comment by dantist90** [comment]
So may be to add such an opportunity in the AR.JS if ARToolKit supports it?

**Comment by nicolocarpignoli** [comment]
ArtoolKit does not support it. There is work to try to port on Artoolkit a stable NFT and then port that version of artoolkit on AR.js.

See: https://github.com/jeromeetienne/AR.js/issues/544

**Comment by umutto** [comment]
Sorry to bump an old issue, but just seen this. I've used white-bordered markers (literally inverse colors of a normal barcode marker) on a different project with a modified version of AR.js 1.6.2. 

I needed to use it for barcodes on black desks, black border markers didn't work without a white buffer zone outside the borders and that didn't seem pretty.

Modifications were minimal and should work with the current version without breaking anything. I can update it for the current and create a pull request some time this week if it would be beneficial.

Here is a sample white bordered barcode marker for reference (I've added a black buffer zone for github, it's not needed):

![000](https://user-images.githubusercontent.com/23720304/72737866-c233a180-3be3-11ea-8f08-9b87ea3fc2bd.png)
4x4_BCH_13_9_3 Barcode marker with value 0
